libsrc/leddevice/LedDeviceAdalightApa102.cpp

```diff
@@ -43,7 +35,7 @@ int LedDeviceAdalightApa102::write(const std::vector<ColorRgb> & ledValues)
 		_ledBuffer[5] = _ledBuffer[3] ^ _ledBuffer[4] ^ 0x55; // Checksum
 	}
 
-	for (unsigned iLed=1; iLed<=ledCount; iLed++) {
+	for (signed iLed=1; iLed<=_ledCount; iLed++) {
```

Comment:
Ah, that makes sense.

On Aug 14, 2016 15:45, "penfold42" <notifications@github.com> wrote:

> In libsrc/leddevice/LedDeviceAdalightApa102.cpp
> <https://github.com/hyperion-project/hyperion.ng/pull/174#discussion_r74701923>
> :
>
> > @@ -43,7 +35,7 @@ int LedDeviceAdalightApa102::write(const std::vector<ColorRgb> & ledValues)
> >  		_ledBuffer[5] = _ledBuffer[3] ^ _ledBuffer[4] ^ 0x55; // Checksum
> >  	}
> >
> > -	for (unsigned iLed=1; iLed<=ledCount; iLed++) {
> > +	for (signed iLed=1; iLed<=_ledCount; iLed++) {
>
> If _ledcount is signed (maybe -1 means something in some context) the
> compiler will throw a warning
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/hyperion-project/hyperion.ng/pull/174/files/703db5dc0a854ae5e00d6caf66ad4d7b854d0f34#r74701923>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAQg67-JdX5NX5-8Agn1KERrGgfM9XInks5qfxvbgaJpZM4JjwFF>
> .
>